### PR TITLE
Test-info en installatiemethode

### PR DESCRIPTION
**Pull request by marjoleintamis**
Info abacus-test.nl en installatiemethode toegevoegd

_What's the scope of the changes?_

- Info over besturingssystemen aangepast (boven methode 1)
- Methode 1 toegevoegd, nummering andere methoden gewijzigd
- Methode 2 bijgewerkt
- Info over Help toegevoegd
- Plaatjes gewijzigd

_What did you test? Which edge cases did you explicitly take into account?_

n.v.t.

_Are there things you want reviewers to definitely take a look at?_

- Even checken of de installatiemethoden ook op jouw computer zo werken, of dat er misschien afwijkingen zijn. 
- Eventueel gekke bewoordingen of onlogische volgorde van de tekst.

_Are there specific people you want feedback from?_

n.v.t.

_Do you have tips on how to test? (For example: data setup, triggering specific errors, etc.)_

Je kunt de instructies volgen en kijken of je ergens vastloopt als je ze op de letter volgt.

### CODE REVIEW

File: documentatie/gebruikersdocumentatie/installeren-en-starten.md [hunk 1]
```diff
@@ -1,17 +1,56 @@
-# Abacus installeren en starten
+# Abacus testen, installeren en starten
 
-Het installeren en starten van Abacus kan op verschillende manieren, met elk verschillende doeleinden.
+Het testen, installeren en starten van Abacus kan op verschillende manieren, met elk verschillende doeleinden.
 Hieronder vind je instructies voor de verschillende methodes.
 
-## Linux en macOS
+## Testen
```
Comment (jschuurk-kr) [inline documentatie/gebruikersdocumentatie/installeren-en-starten.md:6]: Als we het over testen gaan hebben, dan moeten we ook ergens de gautomtatiseerde tests vermelden.

Moet misschien niet meer zijn dan: als je onze geautomatiseerde tests wil draaien, moet je de repo klonen of downloaden en dan in de READMEs van de frontend en backend kijken. En misschien moet deze "Testen" sectie voor geautomatiseerde tests alleen verwijzen naar "Methode 4: Handmatig starten" en zetten we daar een paar zinnen neer.

Kortom, vind vooral dat we het ergens moeten vermelden, want de mensen die de tests willen draaien, zouden dat zonder hulp moeten kunnen, want die klonen de repo, kijken naar de READMEs, etc.

File: documentatie/gebruikersdocumentatie/img/binary-download-1.png
Comment (praseodym) [inline documentatie/gebruikersdocumentatie/img/binary-download-1.png:1]: Aan de rechterkant meteen op 'Releases' klikken scheelt een stap

File: documentatie/gebruikersdocumentatie/installeren-en-starten.md (hunk 1 above)
Comment (praseodym) [reply documentatie/gebruikersdocumentatie/installeren-en-starten.md:6]: Misschien dit kopje 'Uitproberen' noemen?

File: documentatie/gebruikersdocumentatie/installeren-en-starten.md [hunk 2]
```diff
@@ -1,17 +1,56 @@
...
+-->
+
+## Installeren en starten
+
+### Methode 1: installatiebestand downloaden en starten (Linux, macOS, Windows)
+
+Ga in de Abacus-repository naar [Tags](https://github.com/kiesraad/abacus/tags).
 
-De methodes 3 en 4 werken ook in Windows, maar op dit moment is dit meer bedoeld voor developers. Via methode 3 kun je Abacus in Windows zelf gebruiken, en met methode 4 (Docker Desktop) draait Abacus op een virtual machine met Linux. Je kunt nu nog geen installatiebestand voor Windows downloaden, maar we zijn hiermee bezig. Zodra deze optie er is, vind je hier meer info.
+![Installatiebestand downloaden 1](/documentatie/gebruikersdocumentatie/img/binary-download-1.png)
```
[the remark this answers is not in the document]
Comment (marjoleintamis) [reply documentatie/gebruikersdocumentatie/installeren-en-starten.md:35]: Fair point, ik kan die plaatjes wel weghalen inderdaad :)

File: documentatie/gebruikersdocumentatie/installeren-en-starten.md [hunk 3]
```diff
@@ -1,50 +1,122 @@
...
 Hieronder vind je instructies voor de verschillende methodes.
 
-## Linux en macOS
+## Uitproberen
 
-De onderstaande methodes werken waarschijnlijk op alle gangbare (recente) Linux-distributies en zijn getest op Ubuntu (22.04 en hoger) en Debian (12/bookworm). Methode 1 werkt niet op macOS omdat we geen macOS-specifieke builds maken, maar de andere methodes werken wel.
+Als je Abacus wilt uitproberen of testen zonder het te installeren, kun je dit doen op <https://abacus-test.nl/>. Hier heb je de volgende opties:
 
-## Windows
+- Maak een nieuwe omgeving aan door bovenaan op `Create new` te klikken.
```
Comment (jschuurk-kr) [inline documentatie/gebruikersdocumentatie/installeren-en-starten.md:10]: Dit zijn alleen onze wekelijkse release builds. Denk dat het handig is om dat te vermelden.

File: documentatie/gebruikersdocumentatie/installeren-en-starten.md [hunk 4]
```diff
@@ -1,50 +1,122 @@
...
+
+```
+
+Zie ook de [help](#help).
+
+Na het starten zie je een popup van Windows Security over de Windows Firewall. Het maakt niet uit wat je hier selecteert, de omgeving werkt altijd. Klik dus gerust op **Cancel (Annuleren)**.
+
+**Let op:** Het is natuurlijk ook mogelijk om Abacus zonder argumenten te starten, maar als je dat doet bevat de app geen data en zie je op de pagina het bericht *'Verkiezingen niet gevonden'*.
+
+Wanneer Abacus draait, ga je in je browser naar <http://127.0.0.1:8080> om de omgeving te gebruiken.
```
Comment (jschuurk-kr) [inline documentatie/gebruikersdocumentatie/installeren-en-starten.md:70]: Dit geldt voor alle drie de OS-en, maar vanwege de layout lijkt het vooral bij Windows te horen.

File: documentatie/gebruikersdocumentatie/installeren-en-starten.md [hunk 5]
```diff
@@ -1,50 +1,122 @@
...
+
+### Methode 2: build artifact downloaden en starten (Linux, macOS, Windows)
+
+Je kunt ook zelf een bepaalde build downloaden en starten. Voer hiervoor de volgende stappen uit:
 
 1. Ga in de Abacus-repository naar [Actions](https://github.com/kiesraad/abacus/actions).
 2. Klik aan de linkerkant op [Build, lint & test](https://github.com/kiesraad/abacus/actions/workflows/build-lint-test.yml).
 3. Klik op de eerste workflow waarbij de tweede kolom leeg is.
```
Comment (jschuurk-kr) [inline documentatie/gebruikersdocumentatie/installeren-en-starten.md:78]: Build, lint & test draait ondertussen ook op `main` dus het is mogelijk daarop te selecteren / filteren:
image

File: documentatie/gebruikersdocumentatie/installeren-en-starten.md [hunk 6]
```diff
@@ -1,50 +1,122 @@
...
+
+Na het starten zie je een popup van Windows Security over de Windows Firewall. Het maakt niet uit wat je hier selecteert, de omgeving werkt altijd. Klik dus gerust op **Cancel (Annuleren)**.
+
 Het is natuurlijk ook mogelijk om Abacus zonder argumenten te starten, maar als je dat doet bevat de app geen data en zie je op de pagina het bericht *'Verkiezingen niet gevonden'*.
 
-Wanneer de API draait, klik je op de link van de API-server (<http://0.0.0.0:8080>) om naar de browser te gaan en Abacus te gebruiken.
+Wanneer Abacus draait, ga je in je browser naar <http://127.0.0.1:8080> om de omgeving te gebruiken.
 
-### Methode 2: script `pull-and-run` uitvoeren
+### Methode 3: script `pull-and-run` uitvoeren (Linux, macOS)
```
Comment (jschuurk-kr) [inline documentatie/gebruikersdocumentatie/installeren-en-starten.md:119]: In deze sectie staat dat dit script "een productiebuild" bouwt. Ik weet niet of dit klopt, omdat we `cargo run` (denk ik) niet met het `release` profile draaien. Het resultaat van dit script is dus niet hetzelfde als wat we uiteindelijk gaan distribueren voor productie.

@praseodym Kun je zeggen of bovenstaande klopt?

File: documentatie/gebruikersdocumentatie/installeren-en-starten.md [hunk 7]
```diff
@@ -76,16 +148,32 @@ cargo install sqlx-cli
 ./pull-and-run
 ```
 
-### Methode 3: handmatig starten
+### Methode 4: handmatig starten (Linux, macOS, Windows)
 
 *Let op: voor deze methode moet je de repository klonen of downloaden.*
 
-In plaats van het script kun je ook handmatig `cargo run` vanuit de backend-map en `npm run dev` vanuit de frontend-map in twee verschillende terminals starten. Hiervoor gelden dezelfde vereisten als bij methode 2.
+In plaats van het script kun je ook handmatig `cargo run` vanuit de backend-map en `npm run dev` vanuit de frontend-map in twee verschillende terminals starten. Hiervoor gelden dezelfde vereisten als bij het uitvoeren van het pull-and-run-script. Deze optie is bedoeld voor development.
```
Comment (jschuurk-kr) [inline documentatie/gebruikersdocumentatie/installeren-en-starten.md:155]:
```suggestion
In plaats van het script kun je ook handmatig `cargo run` vanuit de backend-map en `npm run dev:server` vanuit de frontend-map in twee verschillende terminals starten. Hiervoor gelden dezelfde vereisten als bij het uitvoeren van het pull-and-run-script. Deze optie is bedoeld voor development.
```

Met `npm run dev` draai je tegen de mock service worker. Met `npm run dev:server` tegen de backend.

File: documentatie/gebruikersdocumentatie/installeren-en-starten.md (hunk 6 above)
Comment (praseodym) [reply documentatie/gebruikersdocumentatie/installeren-en-starten.md:119]: Nee, ik zou het vervangen door "bedoeld om snel Abacus te bouwen en starten".

File: documentatie/gebruikersdocumentatie/img/build-artifact-1.jpg
Comment (praseodym) [inline documentatie/gebruikersdocumentatie/img/build-artifact-1.jpg:1]: Tip: screenshots zijn meestal kleiner als PNG-file.

File: documentatie/gebruikersdocumentatie/installeren-en-starten.md [hunk 8]
```diff
@@ -76,16 +148,32 @@ cargo install sqlx-cli
...
+
+```sh
+./path/to/binary/abacus(-OS-version) --help
+```
+
+En op Windows:
+
+```sh
+start path\to\binary\abacus(-windows-version).exe --help
+```
```
Comment (Lionqueen94) [inline documentatie/gebruikersdocumentatie/installeren-en-starten.md:179]: Ik zou aanraden om bij alle paden, net als in dit gedeelte, [] of () te gebruiken om de versie heen, om duidelijk te maken dat dat variabel is.

Comment (marjoleintamis) [reply documentatie/gebruikersdocumentatie/installeren-en-starten.md:179]: Goed punt, ga ik aanpassen!

File: documentatie/gebruikersdocumentatie/installeren-en-starten.md (hunk 6 above)
Comment (marjoleintamis) [reply documentatie/gebruikersdocumentatie/installeren-en-starten.md:120]: Goeie, heb ik veranderd.

File: documentatie/gebruikersdocumentatie/installeren-en-starten.md (hunk 4 above)
Comment (marjoleintamis) [reply documentatie/gebruikersdocumentatie/installeren-en-starten.md]: Ik heb de opmerking nu verplaatst naar het punt waarop de argumenten voor het eerst worden uitgelegd.

File: documentatie/gebruikersdocumentatie/installeren-en-starten.md (hunk 3 above)
Comment (marjoleintamis) [reply documentatie/gebruikersdocumentatie/installeren-en-starten.md]: Toegevoegd :+1:

File: documentatie/gebruikersdocumentatie/img/build-artifact-1.jpg
Comment (marjoleintamis) [reply documentatie/gebruikersdocumentatie/img/build-artifact-1.jpg:1]: Ok ik dacht dat ze al behoorlijk klein waren, had er specifiek op gelet, als er strikte requirements zijn hoor ik dat graag.

Comment (praseodym) [reply documentatie/gebruikersdocumentatie/img/build-artifact-1.jpg:1]: Het was meer voor een volgende keer bedoeld, niet voor nu!

File: documentatie/gebruikersdocumentatie/installeren-en-starten.md (hunk 4 above)
Comment (jschuurk-kr) [reply documentatie/gebruikersdocumentatie/installeren-en-starten.md]: Sorry voor het moeilijk doen, maar nu staat het alleen bij "Linux en macOS" en zelf zou ik als Windows-gebruiker die hele "Linux en macOS"-sectie overslaan en alleen het  "Windows"-gedeelte lezen.

Comment (jschuurk-kr) [reply documentatie/gebruikersdocumentatie/installeren-en-starten.md]: Daar zie ik nog geen commit van?

Comment (marjoleintamis) [reply documentatie/gebruikersdocumentatie/installeren-en-starten.md]: Ik was 'push origin' vergeten in github desktop, sorry!

**Review by jschuurk-kr — APPROVED** [review]
